osu.Game/Beatmaps/LocalCachedBeatmapMetadataSource.cs

```diff
@@ -318,7 +277,11 @@ private bool queryCacheVersion2(SqliteConnection db, BeatmapInfo beatmapInfo, ou
                 FROM `osu_beatmaps` AS `b`
                 JOIN `osu_beatmapsets` AS `s` ON `s`.`beatmapset_id` = `b`.`beatmapset_id`
                 WHERE `b`.`checksum` = @MD5Hash OR `b`.`filename` = @Path
+                AND `b`.`approved` in (1, 2, 4)
```

Comment:
This doesn't actually work because boolean precedence rules :dizzy_face: 

```suggestion
                WHERE (`b`.`checksum` = @MD5Hash OR `b`.`filename` = @Path)
                AND `b`.`approved` in (1, 2, 4)
```

Reply:
I saw that and mentally thought through it and even tested it and it all seemed to check out. Turns out I was wrong.